src/components/Button.js

```diff
@@ -46,26 +58,45 @@ export const Button = ({ disabled, invert, notFullWidth, onPress, title }) => {
       titleStyle={[
         styles.titleStyle,
         invert && styles.titleStyleInvert,
-        needLandscapeStyle && styles.titleStyleLandscape
+        needLandscapeStyle && styles.titleStyleLandscape,
+        extraLarge && styles.extraLargeTitle,
+        large && styles.largeTitle,
+        medium && styles.mediumTitle,
+        small && styles.smallTitle
       ]}
       disabledStyle={styles.buttonStyleDisabled}
       disabledTitleStyle={styles.titleStyle}
       buttonStyle={[
         styles.buttonStyle,
         invert && styles.buttonStyleInvert,
-        isDelete && styles.rejectStyle
+        isDelete && styles.rejectStyle,
+        extraLarge && styles.extraLarge,
+        large && styles.large,
+        medium && styles.medium,
+        small && styles.small
       ]}
       containerStyle={[styles.containerStyle, needLandscapeStyle && styles.containerStyleLandscape]}
       ViewComponent={invert || isDelete || disabled ? undefined : DiagonalGradient}
       useForeground={!invert}
       accessibilityLabel={`${title} ${consts.a11yLabel.button}`}
       disabled={disabled}
+      icon={{
+        name: iconName,
+        type: IconSetName.toLowerCase(),
```

Comment:
i think we do not need `.toLowerCase()` here, as we need to pass the right string as `IconSetName`

Reply:
i have added it as a safety in case we forget to write small in the future

Reply:
that's not necessary. if one adds a string manually, it needs to be added correctly. why should one pass an uppercased name? if this code wuld be generic, ok, but for a manually passed string it is too much.

Reply:
I think this problem has been eliminated with the newly updated icon prop.

https://github.com/smart-village-solutions/smart-village-app-app/commit/5933f8013e0cfaaab2a97c661ecfa3b13eda8857